core-bundle/src/Resources/contao/library/Contao/ArrayUtil.php

```diff
@@ -44,6 +44,19 @@ public static function arrayInsert(&$arrCurrent, $intIndex, $arrNew): void
 		array_splice($arrCurrent, $intIndex, 0, $arrNew);
 	}
 
+	public static function recursiveKeySort(array &$array): void
```

Comment:
Could this also replace this?
 https://github.com/contao/contao/blob/3a1c6e8824322d76a05e188ad7fa0f2fc0ea2aaa/core-bundle/tests/Image/Studio/FigureBuilderIntegrationTest.php#L1681-L1689

Reply:
Sure, done :)